### PR TITLE
⚠️ CONFLICT! Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request: CONFLICT #

Achtung!!!

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-packer.git`](https://github.com/cisagov/skeleton-packer.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

The `lineage/skeleton` branch has **one or more unresolved merge conflicts**
that you must resolve before merging this pull request!

## How to resolve the conflicts ##

1. Take ownership of this pull request by removing any other assignees.

1. Clone the repository locally, and reapply the merge:

    ```console
    git clone git@github.com:cisagov/terraformer-packer.git terraformer-packer
    cd terraformer-packer
    git remote add skeleton https://github.com/cisagov/skeleton-packer.git
    git remote set-url --push skeleton no_push
    git switch develop
    git switch --create lineage/skeleton --track origin/develop
    git pull skeleton HEAD
    git status
    ```

1. Review the changes displayed by the `status` command.  Fix any conflicts and
   possibly incorrect auto-merges.

1. After resolving each of the conflicts, `add` your changes to the
   branch, `commit`, and `push` your changes:

    ```console
    git add README.md ansible/cloud_init.yml ansible/example.yml ansible/terraform.yml ansible/ufw.yml ansible/vnc.yml ansible/xfce.yml src/packer.pkr.hcl src/version.txt 
    git commit
    git push --force --set-upstream origin lineage/skeleton
    ```

    Note that you may *append* to the default merge commit message
    that git creates for you, but *please do not delete the existing
    content*.  It provides useful information about the merge that is
    being performed.

1. Wait for all the automated tests to pass.

1. Confirm each item in the "Pre-approval checklist" below.

1. Remove any of the checklist items that do not apply.

1. Ensure every remaining checkbox has been checked.

1. Mark this draft pull request "Ready for review".

## ✅ Pre-approval checklist ##

- [x] ✌️ The conflicts in this pull request have been resolved.
- [x] *All* future TODOs are captured in issues, which are referenced in code comments.
- [x] All relevant type-of-change labels have been added.
- [x] All relevant repo and/or project documentation has been updated to reflect the changes in this PR.
- [x] Tests have been added and/or modified to cover the changes in this PR.
- [x] All new and existing tests pass.

---

> [!NOTE]
> You are seeing this because one of this repository's maintainers has
> configured [Lineage] to open pull requests.

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 This is a generic skeleton project that can be used to quickly get a
 new [cisagov](https://github.com/cisagov) GitHub
-[Packer](https://packer.io) project started. This skeleton project
+[Packer](https://packer.io) template project started. This skeleton project
```
Comment (dav3r) [inline README.md:7]: This is not a skeleton repo - the verbiage should reflect that.

File: README.md [hunk 2]
```diff
@@ -147,7 +135,7 @@ region_kms_keys = {
 ```
 
 ```console
-AWS_PROFILE=cool-images-ec2amicreate-terraformer-packer packer build --timestamp-ui -var-file release.pkrvars.hcl src/packer.pkr.hcl
+AWS_PROFILE=cool-images-ec2amicreate-terraformer-packer packer build --timestamp-ui -var-file release.pkrvars.hcl packer.pkr.hcl
```
Comment (dav3r) [inline README.md]: Shouldn't `packer.pkr.hcl` be replaced here by `build.pkr.hcl`?

File: README.md [hunk 3]
```diff
@@ -166,6 +154,46 @@ terraform init --upgrade=true
...
+| skip\_create\_ami | Indicate if Packer should not create the AMI. | `bool` | `false` | no |
+
+## Outputs ##
+
+No outputs.
+<!-- END_TF_DOCS -->
+
 ## New Repositories from a Skeleton ##
 
 Please see our [Project Setup guide](https://github.com/cisagov/development-guide/tree/develop/project_setup)
```
Comment (dav3r) [inline README.md:199]: This section should be removed since this is not a skeleton.

**Review by dav3r — CHANGES_REQUESTED** [review]
Please take a look at the items I noted.

File: README.md (hunk 3 above)
Comment (jsf9k) [reply README.md:199]: Yes, I just realized that the `README.md` for this project was never updated from the skeleton boilerplate.  This is taken care of in #57, which can be merged after this PR.

File: README.md (hunk 1 above)
Comment (jsf9k) [reply README.md:7]: Yes, I just realized that the `README.md` for this project was never updated from the skeleton boilerplate.  This is taken care of in #57, which can be merged after this PR.

File: README.md (hunk 2 above)
Comment (jsf9k) [reply README.md]: I think you're right, and that change should be made upstream.  If @mcdonnnj agrees then I'll create the PR.

Comment (jsf9k) [reply README.md]: Actually, the `prerelease.yml` and `release.yml` workflows suggest that `packer.pkr.hcl` should be replaced with `.`.

Comment (jsf9k) [reply README.md]: See cisagov/skeleton-packer#384.

Comment (dav3r) [reply README.md]: Ah, right, that makes sense! 👍

Comment (dav3r) [reply README.md]: I will re-review once https://github.com/cisagov/skeleton-packer/pull/384 flows into this PR.

**Review by dv4harr10 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (jsf9k) [reply README.md]: See commit e1675b8.

**Review by dav3r — APPROVED** [review]
LGTM 👍

**Review by mcdonnnj — APPROVED** [review]
LGTM ✔

**Review by jsf9k — APPROVED** [review]

**Pull request merged**